dev-tools/mage/crossbuild.go

```diff
@@ -294,7 +294,7 @@ func (b GolangCrossBuilder) Build() error {
 	// container. So we ensure that all path separators are `/`.
 	buildCmd = filepath.ToSlash(buildCmd)
 
-	dockerRun := sh.RunCmd("docker", "run")
+	dockerRun := sh.RunCmd("docker", "run", "--quiet")
```

Comment:
According to the docs:

```bash
$ docker run --help
...
      --pull string                      Pull image before running ("always", "missing", "never") (default "missing")
      -q, --quiet                            Suppress the pull output
```

See https://docs.docker.com/reference/cli/docker/container/run/

<img width="679" alt="image" src="https://github.com/user-attachments/assets/eee7b60e-74c5-4459-b372-109fa4813b82" />